cedar-policy-core/src/ast/policy_set.rs

```diff
@@ -457,11 +457,16 @@ impl PolicySet {
         self.templates.is_empty() && self.links.is_empty()
     }
 
-    /// Lookup a template by policy id
+    /// Lookup a template by policy id, returns [`Option<Arc<Template>>`]
     pub fn get_template(&self, id: &PolicyID) -> Option<Arc<Template>> {
         self.templates.get(id).cloned()
     }
 
+    /// Lookup a template by policy id, returns [`Option<&Template>`]
+    pub fn get_template_ref(&self, id: &PolicyID) -> Option<&Template> {
+        self.templates.get(id).map(AsRef::as_ref)
+    }
```

Comment:
I think in other places we call these `get_template_arc` and just `get_template`.  Should we change the other places to match this convention?

Reply:
I think `get_template_arc` and `get_template` make more sense to me.